src/Network/Network/Generated/LoadBalancer/NewAzureRMLoadBalancerCommand.cs

```diff
@@ -192,6 +202,15 @@ public override void Execute()
                 vSku.Name = this.Sku;
             }
 
+            // Tier
+            if (this.Tier != null)
+            {
+                if (vSku == null)
```

Comment:
we can't have tier without sku right?

Reply:
Yep, I think this is a question of where we want to error. If we don't give values for sku name or tier they default to basic/regional. The only time vSku will be null here is when someone doesn't specify sku name (so defaults to basic) and then they specify a sku tier. If they specify the tier as global, this won't work with a default sku of basic which arm will tell them. But if we choose not to instantiate the vSku object here, we'll get an error for accessing an un-instantiated object instead, which is less informative. This also will error in the case that they don't specify sku name but they specify that tier is regional and while this would be a bit redundant, I don't think we should error in this case. I could add code to check here and produce a custom error if they are instantiating to global without sku specified, but because arm will already catch this anyway, I don't think it's entirely necessary.